ivy/functional/ivy/general.py

```diff
@@ -1395,10 +1395,10 @@ def shape_to_tuple(shape: Union[int, Tuple[int], List[int]]):
     """
     if ivy.is_array(shape):
         raise Exception("shape_to_tuple does not accept arrays as input")
-    if isinstance(shape, int):
+    if isinstance(shape, tuple):
         return shape
     else:
-        return tuple(shape)
+        return (shape,)
```

Comment:
we actually **do** want to explicitly cast to `tuple` here, rather than place the existing `shape` instance into a `tuple`. Backend-specific shape classes will in general support casting to `tuple`. If the shape is an `int`, it doesn't _really_ matter whether we return a `tuple` or not, but we can in this case change to `return (shape,)` rather than `return shape` if an `int` is detected, and I don't think this will break anything, and is in fact probably better given the name of the function.